M2/Macaulay2/packages/ResolutionsOfStanleyReisnerRings.m2

```diff
@@ -0,0 +1,592 @@
+            ComplexIncludingMinElt = barycentricSubdivision D;
+            R = ring(ComplexIncludingMinElt);
+            B = faceDelete(R_0,ComplexIncludingMinElt);
+            kSOP(B,n)
+        Text
+            Also included is the option to input any simplicial complex {\tt D} in order to obtain the colorful system of parameters for the face ring of the corresponding barycentric subdivision of {\tt D}.
+        Example
+            S = QQ[a..e];
+            D = simplicialComplex {a*b*c,c*d,e};
+            kSOP(D)
+    Caveat
+         This current version requires the input of the barycentric subdivision of a simplicial complex {\tt D} without the minimal element of the face poset, which corresponds to the empty set of {\tt D}. This must be acomplished by using using the {\tt barycentricSubdivision} method, included in this package. It also has the option of inputting any simplicial complex. The system then takes the barycentric subdivision of {\tt D} and removes its minimal element in order to obtain the colorful system of parameters. 
+    SeeAlso
+        barycentricSubdivision
+        SimplicialComplexes
+        Posets 
+        SimplicialDecomposability
+///
+
+doc ///
+    Key
+        uSOP
+    Headline
+        Universal System of Parameters (USOP)
+    Usage 
+        uSOP D
+        uSOP F
+    Inputs
+        D:SimplicialComplex
+        F:List
+            a list of facets defining a simplicial complex
+    Outputs
+        Theta:List
+         a list of elements from the Stanley-Reisner ring of {\tt D}
+    Description
+        Text
+            The following example uses the simplicial complex consisting of a triangle of dimension 2 attached to an edge and a isolated vertex.
+        Example
+            S = ZZ/2[a..e];
+            F = {a*b*c,c*d,e}
+            D = simplicialComplex F
+            uSOP D
+            uSOP F
+    SeeAlso
+        SimplicialComplexes
+/// 
+
+doc ///
+      Key
+        sumBetti
+    Headline
+        Computes the sum of Betti numbers.
+    Usage
+        sumBetti M
+    Inputs
+        M:Module
+    Outputs
+         :ZZ
+             sum of the Betti numbers of the resolution of {\tt M} 
+    Description 
+        Example
+            S = QQ[a..f];
+            m = matrix{{a,b,d,e},{b,c,e,f}} 
+            M = coker m
+            sumBetti M
+    SeeAlso
+        Module 
+///
+
+doc ///
+      Key
+        equalBettiTally
+    Headline
+        Computes the Betti table of two modules and compares for equality 
+    Usage
+        equalBettiTally(M,N)
+    Inputs
+        M:Module
+        N:Module
+    Outputs
+        :Boolean
+    Description 
+        Example
+            S = QQ[a..f];
+            m = matrix{{a,b,d,e},{b,c,e,f}} 
+            M = coker m
+            n = matrix{{a,b},{c,d},{e,f}}
+            N = coker n
+            equalBettiTally(M,N)
+    SeeAlso
+        Module 
+///
+
+doc ///
+      Key
+        universalPresentation 
```

Comment:
add keys for universalPresentation(SimplicialComplex) and universalPresentation(List)